### DISCUSSION TITLE
Caduceus: Queue based web-hook Instead of HTTP web-hook

### DISCUSSION THREAD

**Discussion by Sachin4403** [xmidt-org/xmidt#75] [Ideas]
Hi Team,

As of today we have HTTP Webhooks in Xmidt which is registered by Tr1d1um and later used by Caduceus, and in Caduceus every web-hook has an inbuild queue which is used to queue the messages.  Now as a part of this Idea, I would like to replace that web-hook queue with an external queue-based system so that none of the messages will be lost caduceus will be doing the self-heal, and on the other hand consumers(URL of the web-hook) won't be bombarded with spikes.

cc: @schmidtw

**Comment by schmidtw** [comment]
We are definitely open to supporting multiple delivery methods.
